src/lib/Scenes/Home/Components/ForYou/Components/ArtworkRail.tsx

```diff
@@ -0,0 +1,361 @@
+      return
+    }
+
+    if (this.hasAdditionalContent()) {
+      return (
+        <TouchableHighlight style={styles.viewAllButton} onPress={this.handleViewAll} underlayColor={"gray"}>
+          <Text style={styles.viewAllText}>VIEW ALL</Text>
+        </TouchableHighlight>
+      )
+    }
+
+    // otherwise, use a spacer view
+    return <View style={{ height: 30 }} />
+  }
+
+  renderExpansionButton() {
+    if (this.expandable() && !this.state.expanded) {
+      return (
+        <TouchableHighlight style={styles.expansionButton} onPress={this.expand} underlayColor={"white"}>
+          <Image style={{ height: 8, width: 15, alignSelf: "center", resizeMode: "center" }} source={chevron} />
+        </TouchableHighlight>
+      )
+    }
+  }
+
+  renderModuleResults() {
+    if (this.props.rail.results && this.props.rail.results.length) {
+      return (
+        <View style={[styles.container, { height: this.mainContainerHeight() }]}>
+          {this.renderGrid()}
+          {this.renderViewAllButton()}
+          <Separator />
+          {this.renderExpansionButton()}
+        </View>
+      )
+    } else {
+      return <Spinner style={{ flex: 1 }} />
+    }
+  }
+
+  railStyle() {
+    const sideMargin = isPad ? 40 : 20
+    const style: any = { marginLeft: sideMargin, marginRight: sideMargin }
+
+    if (!this.state.didPerformFetch) {
+      style.minHeight = minRailHeight
+    }
+
+    return style
+  }
+
+  render() {
+    if (this.state.loadFailed) {
+      return null
+    }
+
+    const hasArtworks = this.props.rail.results && this.props.rail.results.length
+    if (this.state.didPerformFetch && !hasArtworks) {
+      // if the data has been fetched but there are no results, hide the whole thing
+      return null
+    }
+
+    return (
+      <View accessibilityLabel="Artwork Rail" style={{ paddingBottom: this.state.expanded ? 0 : 12 }}>
+        <ArtworkRailHeader rail={this.props.rail} handleViewAll={this.handleViewAll} />
+        <View style={this.railStyle()}>
+          {this.renderModuleResults()}
+        </View>
+      </View>
+    )
+  }
+}
+
+interface Styles {
+  container: ViewStyle
+  gridContainer: ViewStyle
+  expansionButton: ViewStyle
+  viewAllButton: ViewStyle
+  viewAllText: TextStyle
+}
+
+const styles = StyleSheet.create<Styles>({
+  container: {
+    backgroundColor: "white",
+  },
+  gridContainer: {
+    overflow: "hidden",
+  },
+  expansionButton: {
+    height: 40,
+    width: 40,
+    backgroundColor: "white",
+    borderColor: colors["gray-regular"],
+    borderWidth: 1,
+    borderRadius: 30,
+    alignSelf: "center",
+    top: -20,
+    justifyContent: "center",
+  },
+  viewAllButton: {
+    width: 240,
+    height: 40,
+    backgroundColor: "black",
+    alignSelf: "center",
+    justifyContent: "center",
+    marginBottom: 30,
+    marginTop: 30,
+  },
+  viewAllText: {
+    color: "white",
+    textAlign: "center",
+    fontSize: 14,
+    fontFamily: "Avant Garde Gothic ITCW01Dm",
```

Comment:
This font's name has changed; you can import `data/fonts` and then use `fonts["avant-garde-regular"]` here so it stays consistent.